src/main/java/cs361/battleships/models/Board.java

```diff
@@ -11,6 +11,7 @@
 	@JsonProperty private List<Ship> ships;
 	@JsonProperty private List<Result> attacks;
 	@JsonProperty private List<Result> blockedShips;
+	@JsonProperty private List <Ship> submergedSquares;
```

Comment:
Why is submerged squares here? It seems like it should be in the ship or squares class instead. It also doesn't seem to be used in this class unless I'm missing something